plasmapy/__init__.py

```diff
@@ -15,5 +15,5 @@
 import sys
 import warnings
 
-if sys.version_info[:2] < (3, 6):  # coveralls: ignore
+if sys.version_info[:2] < (3, 6):  # pragma: no cover
```

Comment:
Oh. I seem to have been editing those files in parallel. I'll do some rebasing here and limit my changes to Species and minor changes in Plasma.